release-notes/7.0/supported-os.md

```diff
@@ -31,7 +31,7 @@ OS                                    | Version               | Architectures
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
 [CentOS][CentOS]                      | 7+                    | x64               | [CentOS][CentOS-lifecycle]
 [Debian][Debian]                      | 10+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
-[Fedora][Fedora]                      | 33+                   | x64               | [Fedora][Fedora-lifecycle]
+[Fedora][Fedora]                      | 35+                   | x64               | [Fedora][Fedora-lifecycle]
```

Comment:
For what it's worth, the .NET distro packages in Fedora include `arm64` and `s390x` support in addition to `x64`

Reply:
@omajid  --- Can you check https://github.com/dotnet/core/pull/9294?